src/wp-admin/js/editor.js

```diff
@@ -382,11 +378,7 @@ window.wp = window.wp || {};
 		}
 
 		/**
-<<<<<<< HEAD:src/wp-admin/js/editor.js
-		 * @summary Get adjusted selection cursor positions according to HTML tags/shortcodes
-=======
-		 * Gets adjusted selection cursor positions according to HTML tags, comments, and shortcodes.
```

Comment:
Shouldn't you keep the extended explanation?

Reply:
You seem to be looking at the commit with the conflicts still in place, if you look at the final set of changes:
https://github.com/ClassicPress/ClassicPress/pull/990/files
You should see that the updated description has been retained.

Reply:
I click on *Files changed* and see only 1 file and 3 minor changes, none of which include that comment.
![Comment tags may break when switching Editor Mode by mattyrob · Pull Request 990 · ClassicPressClassicPress-04](https://user-images.githubusercontent.com/7957680/165862391-55b54add-913c-4eba-9463-bf71ad05ba05.jpg)

Reply:
Compare the file changes to the upstream Changeset:
https://core.trac.wordpress.org/changeset/44649

We would not usually add the commit messages into files if you are looking at lines like:
`b3a658edbc (TinyMCE: treat HTML comments same as tags when keeping cursor position. Fixes breaking the More and Nextpage tags when switching from Text to Visual.):src/js/_enqueues/wp/editor/base.js`

Reply:
Thanks for explaining what that is. Sorry for my confusion.